src/main/resources/templates/tags/index.html

```diff
@@ -67,7 +67,7 @@
                 <div class="col-4">
                     <a class="btn btn-primary float-right" href="tags/create">Create a Tag</a>
                 </div>
-                <hr width="60%" align="left">
+                <hr width="100%" align="left">
             </div>
             <div class="row">
                 <div class="col-4">
```

Comment:
cols here don't sum to 12. Could also use a little css instead of a <hr>, but that's just my opinion

Reply:
What would you recommend for the CSS, is there something you have in mind or just anything that is better than '<hr>'?

Reply:
I usually go with a class that gives a border bottom or something like that. 
`.mySeperator {
  border-bottom: 1px solid #{hexColorCode or name of common color}
}`
It's more customizable, but if <hr> works then it aint a big deal

Reply:
So it works as is right now, but I think it would be valuable to use the most up to date or effective methods. Do we have a stylesheet currently that I'm missing or should we create one?

Reply:
I actually am not sure of the best methods for linking a stylesheet properly with spring. We had one before that had trouble linking the conventional html way earlier. Feel free to look into it and make a stylesheet if you'd like. We don't currently have one. You could potentially make a fragment (see the header file and how it is included in pages) and simply put css in there as well as the links for bootstrap and jquery. Might be a good idea

Reply:
I will go ahead and make a separate branch for that so that we can get this pull in. It seems to have been waiting for a bit and fixes a few things.